### PR TITLE
feat(reference): Add BOM example

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: reference/halyard/index.md [hunk 1]
```diff
@@ -95,3 +95,51 @@ others.
 You can see what global Service Settings Halyard has generated for your current 
 Deployment of Spinnaker by reading
 `~/.hal/$DEPLOYMENT/history/service-settings.yml`.
+
+### Bill of Materials
+
+Since Spinnaker is composed of several microservices, we need some way to
+express which versions of each service have been validated together, and
+provide that set of service versions a top-level version that can be referred
```
Comment (duftler) [inline reference/halyard/index.md]: s/referred/referenced

File: reference/halyard/index.md [hunk 2]
```diff
@@ -95,3 +95,51 @@ others.
...
+    version: 0.4.0-393
+  rosco:
+    version: 0.3.0-393
+  spinnaker:
+    version: 0.4.0-393
+timestamp: '2017-05-26 11:30:46'
+version: master-2017-05-26-393
+```
+
+If your curious how to deploy Spinnaker yourself using a validated BOM, read
```
Comment (duftler) [inline reference/halyard/index.md]: s/your/you're

File: reference/halyard/index.md [hunk 3]
```diff
@@ -95,3 +95,51 @@ others.
...
+more in the [deployments](/reference/halyard/deploys) reference page.
```
Comment (duftler) [inline reference/halyard/index.md]: Maybe: s/in/on

**Review by duftler — APPROVED** [review]
LGTM after addressing comments.

**Comment by lwander** [comment]
Thanks

**Pull request merged**